pre-docs/bolt_installing.md

```diff
@@ -125,27 +72,49 @@ http://yum.puppet.com/puppet5/ Packages are named using the convention
 the release package for Puppet 5 Platform on Linux 7 is
 puppet5-release-el-7.noarch.rpm.
 
-> Note: These packages require you to download the Puppet 5 Platform and include
-> puppet-agent as a dependency. If you use an earlier puppet agent you'll have
-> to upgrade it.  Download and install the software and its
-> dependencies. Use the commands appropriate to your system.
-
-1. -  Enterprise Linux 6
-      ```
-      sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-6.noarch.rpm
-      sudo yum install bolt
-      ```
-   -  Enterprise Linux 7
-      ```
-      sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-7.noarch.rpm
-      sudo yum install bolt
-      ```
-   -  SUSE Linux Enterprise Server 12
-      ```
-      sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-sles-12.noarch.rpm
-      sudo yum install bolt
-      ```
-2. Run a Bolt command to verify it has installed correctly.
-   ```
-   bolt --help
-   ```
+> Note: These repositories include the Puppet 5 Platform. If you wish to install
+> only the Bolt package you can install the packages directly as well.
+
+- Enterprise Linux 6
+  ```
+  sudo rpm -Uvh  https://yum.puppet.com/puppet5/puppet5-release-el-6.noarch.rpm
+  sudo yum install bolt
+  ```
+- Enterprise Linux 7
+  ```
+  sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-el-7.noarch.rpm
+  sudo yum install bolt
+  ```
+- SUSE Linux Enterprise Server 12
+  ```
+  sudo rpm -Uvh https://yum.puppet.com/puppet5/puppet5-release-sles-12.noarch.rpm
+  sudo zypper install bolt
+  ```
+
+Get started with `bolt --help`.
+
+## Install Bolt as a gem
+
+Install Bolt and any dependencies for your operating system, such as Ruby, a
```

Comment:
We should probably explicitly call out that it depends on Ruby 2.3 and therefore only very modern platforms are supported. Just so otherwise users don't try and fail to follow our instructions.

Reply:
Updated.